src/app/router/Options/Options.tsx

```diff
@@ -12,14 +12,21 @@ import Send from "@screens/Send";
 import Settings from "@screens/Settings";
 import Unlock from "@screens/Unlock";
 import ChooseConnector from "@screens/connectors/ChooseConnector";
+import { useTranslation } from "react-i18next";
 import { HashRouter, Navigate, Outlet, Routes, Route } from "react-router-dom";
 import { ToastContainer } from "react-toastify";
 import Providers from "~/app/context/Providers";
 import RequireAuth from "~/app/router/RequireAuth";
 import getConnectorRoutes from "~/app/router/connectorRoutes";
+import i18n from "~/i18n/i18nConfig";
+import {
+  translationI18nNamespace,
```

Comment:
fyi @im-adithya I forgot about these. Might be good to use thee in the future to help keeping everything aligned. What do you think?